src/DonationForms/V2/Endpoints/ListDonationForms.php

```diff
@@ -128,10 +128,21 @@ public function handleRequest(WP_REST_Request $request): WP_REST_Response
             $this->listTable->items($forms, $this->request->get_param('locale') ?? '');
             $items = $this->listTable->getItems();
 
-            foreach($items as &$item ) {
+            foreach ($items as &$item) {
+                $v2form = defined('GIVE_NEXT_GEN_VERSION') && ! get_post_meta($item['id'], 'formBuilderFields');
+                $migrate = $v2form
+                           && ! get_post_meta($item['id'], 'migratedFormId');
```

Comment:
This _should_ be checking if the v2 form ID is specified as a `migratedFormId` on a v3 form. The v3 form holds the reference to a migrated v2 form.

Reply:
@kjohnson it makes sense that the v3 form holds the reference to the migrated v2 form, but still, we need some flag on the v2 form in order to show/hide migrate and transfer links. Without that, I will have to loop through all v3 forms and collect references from them, and then loop again through all v2 forms and check if the v2 form id exists in the collection. This is not reliable. I suggest that we add a flag on the v2 form as well.